scopes/scope/scope/scope.docs.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-description: A server for hosting components
-labels: ['core aspect']
+description: A collaboration server for independent components
```

Comment:
A collaboration server for components. - No need for independent here

Reply:
done